clients/java/src/main/java/io/camunda/zeebe/client/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @deprecated since 8.6 for removal with 8.8, replaced by {@link io.camunda.client}
+ */
+@Deprecated
+package io.camunda.zeebe.client;
```

Comment:
❓ Do you like this solution ? In this way all the old package is deprecated for the editors

Reply:
It should warn users that the package is deprecated

Reply:
I see. Well, I guess it can't hurt? 🤷